functions/_common/readmeTools.ts

```diff
@@ -229,9 +230,28 @@ function* columnReadmeText(col: CoreColumn) {
     yield ""
 }
 
+function* activeFilterSettings(searchParams: URLSearchParams) {
+    const filterSettings = getGrapherFilters(searchParams)
+    if (filterSettings) {
+        yield ""
+        yield `### Active Filters`
+        yield ""
+        yield `A filtered subset of the full data was downloaded. The following filters were applied:`
+        for (const entry of Object.entries(filterSettings)) {
```

Comment:
If `getGrapherFilters` was typed with `Record<string, string>` then you could do the sugary thing of 

```
for (const [key, val] of Object.entries(filterSettings)) {
```

which is 100% absolutely completely not necessary. I'm just sharing coz I think it's neat 🙂

Reply:
True, this is nicer :)